fe/fe-core/src/main/java/com/starrocks/analysis/DecimalLiteral.java

```diff
@@ -382,7 +418,7 @@ public int getFracValue() {
         return fracPart.intValue();
     }
 
-    public static void checkLiteralOverflow(BigDecimal value, ScalarType scalarType) throws AnalysisException {
+    public static void checkLiteralOverflowInBinaryStyle(BigDecimal value, ScalarType scalarType) throws AnalysisException {
```

Comment:
should comment when should use `checkLiteralOverflowInBinaryStyle`, when should use `checkLiteralOverflowInDecimalStyle `